lib/Twig/Template.php

```diff
@@ -568,4 +569,18 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
 
         return $ret;
     }
+
+    /**
+     * Prepare template context parameters if need.
+     *
+     * @param array $context
+     */
+    private function prepareContextParameters(&$context)
```

Comment:
please don't use references. This is a bad idea as it often makes performance worse because of reference mismatches (and there is a mismatch here). See http://jpauli.github.io/2014/06/27/references-mismatch.html for the explanation